src/main/java/appeng/core/features/registries/grinder/GrinderRecipeManager.java

```diff
@@ -393,4 +393,117 @@ else if( item != other.item )
 		}
 
 	}
+
+	@Override
+	public IGrinderRecipeBuilder builder()
+	{
+		return new Builder();
+	}
+
+	/**
+	 * Internal {@link IInscriberRecipeBuilder} implementation.
+	 * Needs to be adapted to represent a correct {@link IInscriberRecipe}
+	 */
+	private static final class Builder implements IGrinderRecipeBuilder
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a constructor to the class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1258)